sdk/core/azure-core-experimental/src/main/java/module-info.java

```diff
@@ -10,6 +10,11 @@
     exports com.azure.core.experimental.serializer;
     exports com.azure.core.experimental.implementation;
```

Comment:
Why we exported "implementation"?

Reply:
We shouldn't export this. Looks like an oversight and should be removed. Since this is in core-experimental, removing this is not a breaking change.

Reply:
Removed, and added one line in changelog.
```
### Other Changes

- Removed `com.azure.core.experimental.implementation` from "export" clause of module-info.java.
```